user/chrome.md

```diff
@@ -19,3 +20,40 @@ addons:
 ## Headless mode
 
 You can use Google Chrome in [headless mode](/user/gui-and-headless-browsers/#Using-the-Chrome-addon-in-the-headless-mode).
+
+## Sandboxing
+
+For security reasons, Google Chrome is unable to provide sandboxing when it is running in the
+[container-based environment](https://docs.travis-ci.com/user/reference/overview/#Virtualization-environments).
+
+To use Chrome in the container-based environment:
+
+1. Pass `--no-sandbox` when invoking the `chrome` command.
+
+  ```yaml
+  sudo: false
+  addons:
+    chrome: stable
+  script:
+    - chrome --no-sandbox
```

Comment:
I don't see the value in keeping this YAML example, given that I don't think it even works correctly and is not a common use case AFAIK.

Reply:
I agree.